src/components/MenuItem.js

```diff
@@ -45,14 +44,17 @@ const defaultProps = {
     subtitle: undefined,
     iconType: CONST.ICON_TYPE_ICON,
     onPress: () => {},
+    onPressIn: () => {},
+    onPressOut: () => {},
+    onSecondaryInteraction: () => {},
     interactive: true,
     fallbackIcon: Expensicons.FallbackAvatar,
     brickRoadIndicator: '',
     floatRightAvatars: [],
     shouldStackHorizontally: false,
 };
 
-const MenuItem = (props) => {
+const MenuItem = React.forwardRef((props, ref) => {
```

Comment:
This is a purely stylistic change, but using something like [this](https://github.com/Expensify/App/blob/ddcf4209b90c8917fca0eea4f97a19ffe538a0f2/src/components/CheckboxWithLabel.js#L132) is a bit more readable
```js
export default React.forwardRef((props, ref) => (
    // eslint-disable-next-line react/jsx-props-no-spreading
    < MenuItem {...props} forwardedRef={ref} />
));
```